src/routers/auth-router.ts

```diff
@@ -0,0 +1,10 @@
+import { Router } from 'death-adder';
+import authMiddleware from '../middleware/auth-middleware';
+import UsersEndpoint from '../endpoints/users-endpoint';
+
+const authRouter = new Router();
+
+authRouter.addMiddleware(authMiddleware);
+authRouter.addHandler('/users', UsersEndpoint);
```

Comment:
Could you explain why _users_ is a subcomponent of _auth_? Generally I would expect to go to auth to either login, logout, or register. Why is _auth_ also serving a subcomponent of users? Maybe I don't understand your adjectives.